src/main/scala/uk/gov/hmrc/initrepository/Github.scala

```diff
@@ -52,6 +59,7 @@ trait Github {
 
   val IronManApplication = "application/vnd.github.ironman-preview+json"
   val ZzzaxApplication = "application/vnd.github.zzzax-preview+json"
+  val LukeCageApplication = "application/vnd.github.luke-cage-preview+json"
```

Comment:
Where are the docs about their naming strategy? :)

Reply:
Ha! I know - surely copyright issues...?